feature-login/src/main/java/com/agb/feature_login/core/interactor/UserUseCase.kt

```diff
@@ -0,0 +1,26 @@
+package com.agb.feature_login.core.interactor
+
+import com.agb.core.common.Operation
+import com.agb.core.domain.model.User
+import com.agb.feature_login.core.repository.UserRepository
+import javax.inject.Inject
+
+sealed class UserUseCase constructor(protected val repository: UserRepository) {
```

Comment:
I think we don't need use cases, especially with sealed class approach

Reply:
This is example for usage in another modules, can be reviewed. But generally, I'm for use of interactors